lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -29,20 +29,28 @@ export class CardViewTextItemComponent implements OnChanges {
     property: CardViewTextItemModel;
 
     @Input()
-    editable: boolean;
+    editable: boolean = false;
+
+    @Input()
+    displayEmpty: boolean = true;
 
     @ViewChild('editorInput')
     private editorInput: any;
 
     inEdit: boolean = false;
     editedValue: string;
+    errorMessages: string[];
 
     constructor(private cardViewUpdateService: CardViewUpdateService) {}
```

Comment:
all those methods don't have the returned type

Reply:
done